src/components/input/tests/InputConnected.spec.tsx

```diff
@@ -0,0 +1,164 @@
+import { mount } from 'enzyme';
+// tslint:disable-next-line:no-unused-variable
+import * as React from 'react';
+import { inputPossibleProps, inputProps } from './InputTestCommons.spec';
+import { Input, IInputProps } from '../Input';
+import { InputConnected } from '../InputConnected';
+import { IReactVaporState } from '../../../ReactVapor';
+import { TestUtils } from '../../../utils/TestUtils';
+import { Store, Provider } from 'react-redux';
+import { clearState } from '../../../utils/ReduxUtils';
+import { findWhere } from 'underscore';
+
+describe('<InputConnected />', () => {
+  let store: Store<IReactVaporState>;
+
+  beforeEach(() => {
+    store = TestUtils.buildStore();
+  });
+
+  afterEach(() => {
+    store.dispatch(clearState());
+  });
+
+  const mountComponentWithProps = (props: IInputProps) => mount(
+    <Provider store={store}>
+      <InputConnected {...props} />
+    </Provider>,
+    { attachTo: document.getElementById('App') },
+  );
+
+  it('should mount without errors in various props scenarios', () => {
+    expect(() => {
+      mountComponentWithProps({});
```

Comment:
You could set `{}` as the default and send nothing here.

Reply:
done 👍